utils/src/chunked_evaluations.rs

```diff
@@ -0,0 +1,15 @@
+//! This module contains a type [ChunkedEvaluations],
+
+use ark_ff::Field;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+/// This struct contains multiple chunk evaluations.
+#[serde_as]
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(bound = "F: ark_serialize::CanonicalDeserialize + ark_serialize::CanonicalSerialize")]
+pub struct ChunkedEvaluations<F: Field> {
+    /// The chunk evaluations.
+    #[serde_as(as = "Vec<crate::serialization::SerdeAs>")]
+    pub chunks: Vec<F>,
```

Comment:
looks good, I think it'd be good to also keep the size of the chunks to mirror `ChunkedPolynomial`